go.mod

```diff
@@ -1,25 +1,27 @@
 module github.com/centml/simple-sidecar
 
-go 1.21
+go 1.23.0
+
+toolchain go1.23.5
 
 require (
-	k8s.io/api v0.19.15
-	k8s.io/apimachinery v0.19.15
-	sigs.k8s.io/yaml v1.2.0
+	k8s.io/api v0.32.1
```

Comment:
Nit: this is v0.32.1 not v0.31 as in the title. I think the title should be updated so it's not confusing

Reply:
Whoops, thanks